src/runtime/pages/document-driven.vue

```diff
@@ -1,8 +1,20 @@
 <script setup lang="ts">
+import { throwError } from '#app'
+import { createError } from 'h3'
 import { useContent, useContentHead } from '#imports'
 
 const { page } = useContent()
 
+// Page not found
+if (!page.value) {
+  throwError(
+    createError({
```

Comment:
Note: We are deprecating `throwError` in next release. Can you please try with `throw createError({...})` if works with latest rc?